pkg/gornir/gornir.go

```diff
@@ -15,20 +15,25 @@ type Gornir struct {
 
 // Filter filters the hosts in the inventory returning a copy of the current
 // Gornir instance but with only the hosts that passed the filter
-func (g *Gornir) Filter(f FilterFunc) *Gornir {
+func (gr *Gornir) Filter(f FilterFunc) *Gornir {
 	return &Gornir{
-		Inventory: g.Inventory.Filter(g, f),
-		Logger:    g.Logger,
+		Inventory: gr.Inventory.Filter(gr, f),
+		Logger:    gr.Logger,
 	}
 }
 
 // RunS will execute the task over the hosts in the inventory using the given runner.
 // This function will block until all the tasks are completed.
-func (g *Gornir) RunS(title string, runner Runner, task Task) (chan *JobResult, error) {
-	results := make(chan *JobResult, len(g.Inventory.Hosts))
+func (gr *Gornir) RunS(title string, runner Runner, task Task) (chan *JobResult, error) {
```

Comment:
Why not call `RunS`, `RunSync` or else to make it easier for the reader?. Same thing for `RunA`...

Reply:
No particular reason, happy to change.